### PR TITLE
Add prescription john deere

### PR DESCRIPTION
**Pull request by palomacione**

### CODE REVIEW

**Comment by palomacione** [comment]
[[Postman] Add prescription upload endpoints to JD](https://trello.com/c/wlSmwrvP/305-postman-add-prescription-upload-endpoints-to-jd)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -4835,6 +4835,88 @@
...
+													],
+													"path": [
+														"services",
+														"beta",
+														"prescription",
+														"api",
+														"users",
+														"{{leaf_user_id}}",
+														"johnDeere"
+													]
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:4914]: Missing organizationId param here :smile:

File: Leaf API.postman_collection.json [hunk 2]
```diff
@@ -4835,6 +4835,88 @@
...
+														{
+															"key": "file",
+															"description": "Select your prescription file (zip file)",
+															"type": "file",
+															"src": []
+														}
+													]
+												},
+												"url": {
+													"raw": "https://{{leaf_api_url}}/services/beta/prescriptions/api/users/{{leaf_user_id}}/johnDeere",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:4901]: Missing **organizationId** param here :smile: 
I think you have to include this information in tab **Params**, leave it enabled and export the collection

**Review by fabyan-alexander — CHANGES_REQUESTED** [review]
Nice work! Just a missing property in the upload file to JD

File: Leaf API.postman_collection.json (hunk 2 above)
Comment (palomacione) [reply Leaf API.postman_collection.json:4901]: e5c81990ba5b821dc71d8a6f482273da2eeb1b3b

Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json:4901]: We need to add this to the GET endpoint too.

**Review by ricardosantacattarina — CHANGES_REQUESTED** [review]

**Review by fabyan-alexander — APPROVED** [review]

**Review by ricardosantacattarina — APPROVED** [review]

**Pull request merged**